src/__tests__/wait-for-dom-change.js

```diff
@@ -1,6 +1,11 @@
 import {waitForDomChange} from '../wait-for-dom-change'
 import {renderIntoDocument} from './helpers/test-utils'
 
+function importModule() {
```

Comment:
When `jest.resetModules()` or `jest.useFakeTimers()` is called, then that can result is breaking test isolation. So could we change this a bit:

```javascript
import {renderIntoDocument} from './helpers/test-utils'

let waitForDomChange

beforeEach(() => {
  jest.useRealTimers()
  jest.resetModules()
  waitForDomChange = require('../').waitForDomChange
})
```

This is probably how we should do it in the other tests as well

Reply:
Fair point :+1: